spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -40,6 +40,7 @@ object SpdxDeclaredLicenseMapping {
         "3-Clause BSD License" to BSD_3_CLAUSE.toExpression(),
         "3-clause bdsl" to BSD_3_CLAUSE.toExpression(),
         "Affero General Public License (AGPL) v. 3" to AGPL_3_0_ONLY.toExpression(),
+        "APACHEv2" to APACHE_2_0.toExpression(),
```

Comment:
Does it need to go one line further down?

Reply:
It anyway needs to go to `SpdxLicenseAliasMapping`.

Reply:
Thx @sschuberth  for spotting that one, updated my PR to make the change to `SpdxLicenseAliasMapping`.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/2460.